packages/web3-common/src/common_utils.ts

```diff
@@ -0,0 +1,306 @@
+import { keccak256 } from 'ethereum-cryptography/keccak';
+import { validator, isHexStrict, utils as validatorUtils, isAddress } from 'web3-validator';
+import { InvalidAddressError, InvalidBytesError, InvalidNumberError } from './errors';
+import { Address, Bytes, HexString, Numbers } from './types';
+
+const SHA3_EMPTY_BYTES = '0xc5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470';
+
+export const bytesToBuffer = (data: Bytes): Buffer | never => {
```

Comment:
This all change seems to be just to reuse the error codes. For which we already discussed that we will be creating a package `web3-errors` when needed. So it seems now it's the need. @spacesailor24 @luu-alex

Reply:
Its not for reusing error codes but for using same base Error class.